Block/BlockTrait.php

```diff
@@ -156,6 +158,15 @@ protected function isIPRestricted()
         return $this->configHelper->isIPRestricted();
     }
 
+    /**
+     * @param int $customerGroupId
+     * @return bool
+     */
+    public function isCustomerDisabled($customerGroupId)
+    {
+        return in_array($customerGroupId, $this->configHelper->getDisabledCustomerGroups());
```

Comment:
Let's make this function return false if the featureswith is disabled

Reply:
@vitaliyreznikov  I've updated the PR.  Could you please re-check?